API/SSW.Rewards.Application/Services/UserService.cs

```diff
@@ -113,6 +118,12 @@ public async Task<CurrentUserViewModel> GetCurrentUser(CancellationToken cancell
                 await _dbContext.SaveChangesAsync(cancellationToken);
             }
 
+            var points = user.UserAchievements.Sum(ua => ua.Achievement.Value);
```

Comment:
These lines (121 - 125) have been deleted on the branch. Not sure how to get them out of this PR.